taxonomies/ingredients.txt

```diff
@@ -74339,15 +74366,16 @@ ciqual_food_name:fr:Sardine, crue
 # 1553 in 20 @2021-09-15
 
 #processing:en: en:fillet
-#<en:sardine
-#en:sardine fillets
-#de:Sardinenfilets
-#fi:sardiinifilee
-#fr:filets de sardines
-#it:filetti di sardine
-#nl:sardinefilets
-#sk:filety sardiniek
+<en:sardine
+en:sardine fillets
+de:Sardinenfilets
+fi:sardiinifilee
+fr:filets de sardines
+it:filetti di sardine
+nl:sardinefilets
+sk:filety sardiniek
 # 142 in 5 languages @2021-09-21
+# We disabled it because we can catch it in processing, but as a result, it does not inherit the CIQUAL property of sardines
```

Comment:
```suggestion
# We disabled it because we thought we could catch it in processing, but as a result, it does not inherit the CIQUAL property of sardines
```